cmd/kuberhealthy/kuberhealthy.go

```diff
@@ -1215,3 +1234,45 @@ func (k *Kuberhealthy) configureInfluxForwarding() {
 	}
 	k.MetricForwarder = metricClient
 }
+
+func listUnstructuredKHChecks() (*unstructured.UnstructuredList, error){
+
+	var unstructuredList *unstructured.UnstructuredList
+	restConfig, err := clientcmd.BuildConfigFromFlags("", configPath)
+	if err != nil {
+		log.Errorln("Error building rest config for dynamic client:", err)
```

Comment:
I try not to log errors right where I return them.  I try to log errors where I decide what to do about them.  That would mean simply decorating this error and passing it upstream, then adding logging where you decide to `continue` past the entry.